src/bentoml/_internal/marshal/dispatcher.py

```diff
@@ -269,8 +267,6 @@ async def controller(self):
                     "BentoML has detected that a service has a max latency that is likely too low for serving. If many 503 errors are encountered, try raising the 'runner.max_latency' in your BentoML configuration YAML file."
                 )
             logger.debug("Dispatcher optimizer training complete.")
-        except asyncio.CancelledError:
-            raise
```

Comment:
I feel like this should just be a return instead of a raise?

Reply:
If it's a return, only this training process will return and the subsequent ones will continue to run:

https://github.com/bentoml/BentoML/blob/6f7a32df9ca24d3299a3a6c13ca3cd381d5a3680/src/bentoml/_internal/marshal/dispatcher.py#L247-L263